set.mm

```diff
@@ -530290,6 +530299,14 @@ be tempted to generalize this (or a similar) theorem by weakening the
       ( weq wi wal ax12v2 sp com12 impbid ) BCDZAKAEZBFZABCGMKALBHIJ $.
   $}
 
+  ${
+    $d x y $.  $d y ph $.
+    $( One direction of the equivalence in ~ exsb is based on fewer axioms.
+       (Contributed by Wolf Lammen, 2-Mar-2023.) $)
+    wl-lem2-exsb $p |- ( E. y A. x ( x = y -> ph ) -> E. x ph ) $=
+      ( weq wi wal wex ax6ev exim mpi exlimiv ) BCDZAEBFZABGZCMLBGNBCHLABIJK $.
+  $}
```

Comment:
Worth moving to Main. What label ? exsbi ?

Reply:
I chose exsb-1